src/Package/Impl/source.extension.vsixmanifest

```diff
@@ -17,6 +17,7 @@
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.R.Core.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.R.Actions.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.R.Debugger.Engine.dll" />
+    <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.R.Wpf.dll" />
```

Comment:
I would like to have R-specific styles and templates in a separate library. We probably should move controls there as well, but that isn't in this changeset.